parlai/tasks/twitter/build.py

```diff
@@ -5,9 +5,13 @@
 # of patent rights can be found in the PATENTS file in the same directory.
 # Download and build the data if it does not exist.
 
+try:
```

Comment:
wait did u guys (with ilia) change the twitter data itself recently? that's not good practice in general.. we need versions/naming for that

Reply:
#729